spec/integration/puppet_x/puppetlabs/scheduled_task/taskscheduler2_v1task_spec.rb

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env ruby
+require 'spec_helper'
+
+require 'puppet/util/windows/taskscheduler' if Puppet.features.microsoft_windows?
+require 'puppet_x/puppetlabs/scheduled_task/taskscheduler2_v1task' if Puppet.features.microsoft_windows?
+
+def dummy_time_trigger
+  now = Time.now
+  {
+    'flags'                   => 0,
+    'random_minutes_interval' => 0,
+    'end_day'                 => 0,
+    'end_year'                => 0,
+    'minutes_interval'        => 0,
+    'end_month'               => 0,
+    'minutes_duration'        => 0,
+    'start_year'              => now.year,
+    'start_month'             => now.month,
+    'start_day'               => now.day,
+    'start_hour'              => now.hour,
+    'start_minute'            => now.min,
+    'trigger_type'            => Win32::TaskScheduler::ONCE,
+  }
+end
+
+# These integration tests confirm that the tasks created in a V1 scheduled task APi are visible
+# in the V2 API, and that changes in the V2 API will appear in the V1 API.
+
+describe "PuppetX::PuppetLabs::ScheduledTask::TaskScheduler2V1Task", :if => Puppet.features.microsoft_windows? do
+  let(:subjectv1) { Win32::TaskScheduler.new() }
+  let(:subjectv2) { PuppetX::PuppetLabs::ScheduledTask::TaskScheduler2V1Task.new() }
+
+  let(:command) { 'cmd.exe' }
+  let(:arguments_before) { '/c exit 0' }
```

Comment:
Never used

Reply:
Oh yeah...that's because I moved them to before(:all) and `let` variables are not available there.

Reply:
Fixed